### ISSUE TITLE
@veramo/kms-local-react-native: Type 'KeyManagementSystem' is missing the following properties

### ISSUE THREAD

**Issue by RobertoSnap** [uport-project/veramo#655]
**Bug severity**
1

**Describe the bug**
Looks like KeyManagementSystem from @veramo/kms-local-react-native does not implement KeyManager manager requirements. 

Type 'KeyManagementSystem' is missing the following properties from type 'AbstractKeyManagementSystem': sign, sharedSecret

**To Reproduce**
Followed the React native guide: https://veramo.io/docs/react_native_tutorials/react_native_setup_identifers/

**Observed behaviour**
Getting type error: 

`Type 'KeyManagementSystem' is missing the following properties from type 'AbstractKeyManagementSystem': sign, sharedSecret`

![09-08-2021 at 15 48 37](https://user-images.githubusercontent.com/4772930/128716820-9735f2cf-71b5-4bc8-a712-cf5f69d0e330.png)
   
**Expected behaviour**
No type errors

**Versions (please complete the following information):**
    "@veramo/core": "^0.0.42",
    "@veramo/credential-w3c": "^0.0.42",
    "@veramo/data-store": "^0.0.42",
    "@veramo/did-manager": "^0.0.42",
    "@veramo/did-provider-ethr": "^0.0.42",
    "@veramo/did-provider-key": "^0.0.42",
    "@veramo/did-resolver": "^0.0.42",
    "@veramo/key-manager": "^0.0.42",
    "@veramo/kms-local": "^0.0.42",
    "@veramo/kms-local-react-native": "^1.2.0",

 - "react-native": "0.64.1",
 - Node Version v16.6.1

**Comment by mirceanis** [comment]
I think the react-native guide needs a refresh.
`@veramo/kms-local-react-native` is no longer needed. You can use `@veramo/kms-local` instead.

**Comment by mirceanis** [comment]
@RobertoSnap If you're still blocked by this, we recently updated the RN guide: https://veramo.io/docs/react_native_tutorials/react_native_setup_identifers

**Comment by RobertoSnap** [comment]
Not blocked by this, but thanks!  ☺️
